src/Nancy.Testing/Browser.cs

```diff
@@ -199,10 +199,18 @@ public BrowserResponse Put(Url url, Action<BrowserContext> browserContext = null
             return this.HandleRequest("PUT", url, browserContext);
         }
 
+        /// <summary>
+        /// Performs a request of the HTTP <paramref name="method"/>, on the given <paramref name="url"/>, using the
+        /// provided <paramref name="browserContext"/> configuration.
+        /// </summary>
+        /// <param name="method">HTTP method to send the request as.</param>
+        /// <param name="url">The URl of the request.</param>
+        /// <param name="browserContext">An closure for providing browser context for the request.</param>
+        /// <returns>An <see cref="BrowserResponse"/> instance of the executed request.</returns>
         public BrowserResponse HandleRequest(string method, Url url, Action<BrowserContext> browserContext)
         {
             var request =
-                CreateRequest(method, url, browserContext ?? (with => {}));
+                this.CreateRequest(method, url, browserContext ?? (with => {}));

```

Comment:
Oh sorry.. I made `DefaultBrowserContext` static. It's a method and on line 44 it's being used as a method group